src/kernel.cpp

```diff
@@ -95,13 +95,24 @@ bool cvk_kernel::setup_descriptor_sets(
         return false;
     }
 
+    // Make enough space to store all descriptor write structures
+    size_t max_descriptor_writes =
+        m_args.size() + program()->literal_sampler_descs().size() + 1;
```

Comment:
Why +1?

Reply:
Didn't occur to me that POD args would be included in m_args as well :-)